src/reducers/app/reducers.ts

```diff
@@ -121,8 +123,29 @@ export const app = (state = initialState, action) => {
       return { ...state, selectedEntity: action.entity };
     case SET_LOADING:
       return { ...state, isLoading: action.isLoading };
-    case SET_NETWORK:
-      return { ...state, network: action.network };
+    case SET_CONFIG:
+      return { ...state, selectedConfig: action.config };
+    case ADD_CONFIG: {
```

Comment:
Adding another configuration with tezos/mainnet but on a different host creates a conflict where the other cannot be selected